### PR TITLE
remove metrics from lite build to support legacy account parity

### PR DESCRIPTION
**Pull request by metal-messiah**
### Overview

This PR addresses an issue where 1220+ agents have added metrics to the lite agent build, but legacy "lite" customers do not have access to the metrics endpoint, causing 403 network codes.  This is handled beyond the scope of the browser agent team, by assigning entitlements that are enforced at ingest time.

### Related Issue(s)

https://discuss.newrelic.com/t/bam-nr-data-net-post-returns-403-error/161276

### CODE REVIEW

**Comment by metal-messiah** [comment]
Awaiting input from ingest and PM on how we want to proceed

**Comment by metal-messiah** [comment]
I think in general the concern is that all customers are confused when *any* newrelic service that used to work is now randomly (in their perception) having errors. 

The 403 doesn't have any messaging tied to it so the customer just assumes the agent is broken. This is inefficient for everyone from the customer to support to us. 

I got this on ingests radar to add a 403 message at the very least, but further planning is needed. 

This pr would just align the lite loader with the behavior in 1216

File: cdn/agent-aggregator/util/features.js [hunk 1]
```diff
@@ -14,7 +17,7 @@ export const modules = {
   metrics: 'metrics'
```
Comment (cwli24) [inline cdn/agent-aggregator/util/features.js:17]: Minor: can be removed

File: cdn/agent-aggregator/util/features.js [hunk 2]
```diff
@@ -14,7 +17,7 @@ export const modules = {
...
 }
 
-const lite = [modules.pageViewEvent, modules.pageViewTiming, modules.metrics]
+const lite = [modules.pageViewEvent, modules.pageViewTiming] // lite agent cannot currently use metrics endpoint, because legacy lite customers are not provisioned access to that endpoint
 const pro = [...lite, modules.jsErrors, modules.ajax, modules.pageAction, modules.sessionTrace]
```
Comment (cwli24) [inline cdn/agent-aggregator/util/features.js:21]: You'll need to add metrics to Pro, to retain it for Pro & SPA, or they won't have them either.

**Review by cwli24 — CHANGES_REQUESTED** [review]

File: cdn/agent-aggregator/util/features.js (hunk 1 above)
Comment (patrickhousley) [reply cdn/agent-aggregator/util/features.js:17]: It's still used and can't be removed.

File: cdn/agent-loader/lite.js [hunk 3]
```diff
@@ -2,16 +2,13 @@
  * Copyright 2020 New Relic Corporation. All rights reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-import { detectPolyfillFeatures } from './utils/feature-detection'
-const PolyfillFeatures = detectPolyfillFeatures();
```
Comment (patrickhousley) [inline cdn/agent-loader/lite.js:6]: Why were these lines removed?

Comment (metal-messiah) [reply cdn/agent-loader/lite.js:6]: these are used to report supportability metrics

Comment (patrickhousley) [reply cdn/agent-loader/lite.js:6]: Oh

**Comment by patrickhousley** [comment]
Need to add changelog entry. Otherwise, looks good once the unit tests are passing.
